mullvad-relay-selector/src/lib.rs

```diff
@@ -8,8 +8,10 @@ mod relay_selector;
 
 // Re-exports
 pub use error::Error;
+pub use relay_selector::detailer;
+pub use relay_selector::query;
+pub use relay_selector::relays::WireguardConfig;
 pub use relay_selector::{
-    detailer, query, AdditionalRelayConstraints, AdditionalWireguardConstraints, GetRelay,
-    RelaySelector, RuntimeParameters, SelectedBridge, SelectedObfuscator, SelectorConfig,
-    WireguardConfig, RETRY_ORDER,
+    AdditionalRelayConstraints, AdditionalWireguardConstraints, GetRelay, RelaySelector,
+    RuntimeParameters, SelectedBridge, SelectedObfuscator, SelectorConfig, RETRY_ORDER,
 };
```

Comment:
Personally I dislike grouping imports, and as long as stable `rustfmt` doesn't complain I probably won't change it. Please let me know if you completely disagree :blush:

Reply:
I prefer nightly fmt since it is more oppinionated and makes the code look more coherent. In this case it seems weird to group some imports and not others.

Reply:
I don't really care, ran the nightly formatter